File: DataStructureandAlgorithm/Arrays/TwoNumberSum.java
package DataStructureandAlgorithm.Arrays;


class TwoNumberSum{
    public static int[] twoNumberSums(int[] array , int targetsum){
        for(int i = 0; i < array.length -1; i++){
            int firstNum = array[i];
            for(int j = i + 1 ;j < array.length; j++){
                int secondNum = array[j] ;
                if (firstNum + secondNum == targetsum) {
                    return new int[]{firstNum,secondNum};
                }
            }
        }
        return new int[0];
    }
    public static void main(String[] args) {
        int[] array = {3 ,5,-4 ,8, 11,1,-1 ,6};
        int targetsum = 10 ;
        System.out.println("The two numbers that add up to the given sum are: " + twoNumberSums( array, targetsum) );
    }
}

File: DataStructureandAlgorithm/Python/Arrays/TwoNumberSum.py
def twoNumberSum(array , targetSum):
  for i in range(len(array) - 1 ):
        firstNum = array[i]
        for j in range(i + 1, len(array)):
            secondNum = array[j]
            if firstNum + secondNum == targetSum :
                return [firstNum,secondNum]
  return []

def main():
    array = {4 , 5, 5 , 7 , 3, 2 , 8, 6}
    targetSum = 10 
    print('so the the two number equal to target are:')
    print(twoNumberSum(array , targetSum))
if __name__=="__main__":
         main()